sklearn/preprocessing/_data.py

```diff
@@ -552,41 +562,41 @@ class StandardScaler(TransformerMixin, BaseEstimator):
 
     Parameters
     ----------
-    copy : boolean, optional, default True
+    copy : boolean, optional, default=True
```

Comment:
for the subsequent entries, you can remove `optional` and use the python type instead of english one; e.g boolean -> bool, string -> str, etc.